CHANGELOG.md

```
@@ -1,76 +1,79 @@
 # Prowide Core - CHANGELOG
 
-#### 9.5.2 - SNAPSHOT
+#### 10.2.2 - SNAPSHOT
   * Added new `FieldEnum` with all the available field names
   * Code security improvements as per CodeQL recommendations
 
-#### 9.5.1 - June 2024
+#### 10.2.1 - June 2024
   * (PW-1913) Added IBAN validation for Egypt local account structure
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)